sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/implementation/ChangeFeedStartFromPointInTimeImpl.java

```diff
@@ -49,15 +49,17 @@ public void populatePropertyBag() {
 
         super.populatePropertyBag();
 
-        setProperty(
-            this,
-            com.azure.cosmos.implementation.Constants.Properties.CHANGE_FEED_START_FROM_TYPE,
-            ChangeFeedStartFromTypes.POINT_IN_TIME);
-
-        setProperty(
-            this,
-            Constants.Properties.CHANGE_FEED_START_FROM_POINT_IN_TIME_MS,
-            this.pointInTime.toEpochMilli());
+        synchronized(this) {
```

Comment:
Does base class method call need to be "synchronized"/single threaded as well?

Reply:
The base class' ChangeFeedStartFromInternal.populatePropertyBag is a no-op as is its baseclass JsonSerializable's populatePropertyBag method.

Reply:
The base class' ChangeFeedStartFromInternal.populatePropertyBag is a no-op as is its baseclass JsonSerializable's populatePropertyBag method.